scripts/drag-n-bar.js

```diff
@@ -1362,6 +1362,11 @@ H5P.DragNBar.prototype.findNewPoint = function (originX, originY, angle, distanc
  */
  H5P.DragNBar.prototype.createMoveableControlBoxOnElement = function ($element, uniqueClassName) {
   
+  const checkIfIsImage = () => {return $element.find("img").length > 0}
```

Comment:
I suggest using implicit return

```suggestion
  const checkIfIsImage = () => $element.find("img").length > 0;
```